src/core/elrs.c

```diff
@@ -42,7 +43,11 @@ static int record_state;
 static uint32_t record_time = 0;
 static bool headtracking_enabled = false;
 
+uint16_t elrs_osd[HD_VMAX][HD_HMAX];
+static uint16_t elrs_osd_overlay[HD_VMAX][HD_HMAX];
+
 void msp_process_packet();
+static void handleOSD(uint8_t *payload, uint8_t size);
```

Comment:
code style, we are using `snake_case` elsewhere

Reply:
fixed